README.md

```diff
@@ -81,6 +74,20 @@ purposes, use a custom parent domain.
 
 ### GitHub Codespaces
 
+Use [GitHub Codespaces](https://docs.github.com/en/codespaces/about-codespaces/what-are-codespaces) to setup a full Drupal dev environment in a browser.
+
+This project demonstrates a complete Drupal 10 development environment,
+utilizing ddev and Codespaces, through your browser. Refer to
+https://docs.github.com/en/codespaces for usage documentation.
+
+Choose one of the following preconfigured development environments based on
+your needs (clicking on a link launches that Codespace):
+- [Standard Environment (Drupal 11 + Nuxt3 Naked)](https://codespaces.new/drunomics/lupus-decoupled-project?quickstart=1&devcontainer_path=.devcontainer%2Fbase_with_nuxt_naked%2Fdevcontainer.json)
```

Comment:
We don't want to repeat here what is written elsewhere. We can link to what is written elsewhere.

We want to have development instructions here, i.e. the same as we had with Gitpod but with Github codespaces. Some of this is TODO for codespaces, what we can mark there for now. 

Also we probably should update Gitpod to mention that this requires a (Paid) Github flex subscription to work.

Reply:
> We don't want to repeat here what is written elsewhere. We can link to what is written elsewhere.

We had the same for gitpod (it is still there, but I'll remove it - there's no need for duplication).

> We want to have development instructions here, i.e. the same as we had with Gitpod but with Github codespaces. Some of this is TODO for codespaces, what we can mark there for now.

Since we removed the secrets and moved them into the follow-up a codespace with configurable parameters at this point does not exist. We should add it to the README.md once we have it.

> Also we probably should update Gitpod to mention that this requires a (Paid) Github flex subscription to work.

Added.